src/core.js

```diff
@@ -2398,15 +2344,31 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
   };
 
   /**
-   * Remove one or more rows from the cell meta object.
+   * Removes or adds one or more rows of the cell meta objects to the cell meta collections.
    *
    * @since 0.30.0
-   * @param {Number} index An integer that specifies at what position to add/remove items, Use negative values to specify the position from the end of the array.
-   * @param {Number} deleteAmount The number of items to be removed. If set to 0, no items will be removed.
-   * @param {Array} items The new items to be added to the array.
+   * @memberof Core#
+   * @function spliceCellsMeta
+   * @param {Number} visualIndex A visual index that specifies at what position to add/remove items.
+   * @param {Number} [deleteAmount=0] The number of items to be removed. If set to 0, no cell meta objects will be removed.
+   * @param {...Object} [cellMetaRows] The new cell meta row objects to be added to the cell meta collection.
    */
-  this.spliceCellsMeta = function(index, deleteAmount, ...items) {
-    priv.cellSettings.splice(index, deleteAmount, ...items);
+  this.spliceCellsMeta = function(visualIndex, deleteAmount = 0, ...cellMetaRows) {
```

Comment:
I would suggest names: `rowsCellMetas` -> `rowCellMetas` -> `cellMeta`. For me it would be more clear.

Reply:
I think that the `cellMetaRows` name, especially with postfix `Rows`, is descriptive enough. To make a body of the method more readability I'll change the argument name for https://github.com/handsontable/handsontable/pull/6254/files#diff-7eb52b366866677666470e019283c8eaR2366.

Reply:
Fixed within https://github.com/handsontable/handsontable/pull/6254/commits/eb1f3258945f22e65a1255460a62cc3da680e58e#diff-7eb52b366866677666470e019283c8eaR2369